lambda/rotator-snowflake/setup/setup.go

```diff
@@ -2,42 +2,130 @@ package setup
 
```

Comment:
This file has the biggest changes--I moved a lot of the configuration/setup logic here so we don't get circular imports.